src/main/java/eu/xenit/gradle/alfresco/DockerAlfrescoExtension.java

```diff
@@ -4,13 +4,16 @@
 import groovy.lang.Closure;
 import java.util.function.Supplier;
 import org.gradle.api.Action;
+import org.gradle.api.GradleException;
 import org.gradle.api.Project;
 
 /**
  * Created by thijs on 9/21/16.
  */
 public class DockerAlfrescoExtension {
 
+    public static final String MESSAGE_BASE_IMAGE_NOT_SET = "Base image not set. You need to configure your base image in the 'dockerAlfresco' extension block";
```

Comment:
This variable will be exposed on the `dockerAlfresco` gradle extension.

Not duplicating the string is good, but I'm not sure if we want to place it right here as a public constant, as users of the plugin are not supposed to use it.

Reply:
Lots of things are public, and therefor usable.
How would you solve this then?